mubench.reviewsite/src/Model/Review.php

```diff
@@ -45,6 +45,21 @@ public function getHitViolationTypes($rank){
         return $this->getReviewFinding($rank)['violation_types'];
     }
 
+    public function hasPositiveHit()
```

Comment:
We have the two concepts "potential hit" and "hit". Please rename to `identifiesHit`.